Team8923_PowerPlay/src/main/java/org/firstinspires/ftc/team8923_PowerPlay/Constants.java

```diff
@@ -0,0 +1,29 @@
+package org.firstinspires.ftc.team8923_PowerPlay;
+
+public class Constants {
+    //ratios
+    static final double TICKS_PER_REVOLUTION = 537.6; // Neverest orbital 20, 7 pulse per revolution
```

Comment:
Do you know what type of motor you're using?

Reply:
They should be the same motors from last year. I'll double check when I'm in the shop

Reply:
they are orbital 20s